src/autopas/baseFunctors/CellFunctor3B.h

```diff
@@ -0,0 +1,643 @@
+/**
+ * @file CellFunctor3B.h
+ * @author M. Muehlhaeusser
+ * @date 25/07/23
+ */
+
+#pragma once
+
+#include "autopas/cells/SortedCellView.h"
+#include "autopas/options/DataLayoutOption.h"
+#include "autopas/utils/ExceptionHandler.h"
+
+namespace autopas::internal {
+/**
+ * A cell functor. This functor is built from the normal Functor of the template
+ * type ParticleFunctor. It is an internal object to handle interactions between
+ * two cells of particles.
+ * @tparam Particle
+ * @tparam ParticleCell
+ * @tparam ParticleFunctor the functor which is used for particle interactions
+ * @tparam DataLayout the DataLayout to be used
+ * @tparam useNewton3
+ * @tparam bidirectional if no newton3 is used processCellPair(cell1, cell2) should also handle processCellPair(cell2,
+ * cell1)
+ */
+template <class Particle, class ParticleCell, class ParticleFunctor, DataLayoutOption::Value DataLayout,
+          bool useNewton3 = false, bool bidirectional = true>
+class CellFunctor3B {
+ public:
+  /**
+   * The constructor of CellFunctor3B.
+   * @param f The ParticleFunctor which should be used for the interaction.
+   * @param sortingCutoff This parameter indicates the maximal distance the sorted particles are to interact. This
+   * parameter is only relevant for optimization (sorting). This parameter normally should be the cutoff, for building
```

Comment:
In practice, we always pass cutoff+skin, but for example in the LCC01 traversal we have also listed the todo that we should pass only cutoff for these traversals:
https://github.com/AutoPas/AutoPas/blob/8bbbd30c40664ababa62f2618cf90072219588e2/src/autopas/containers/linkedCells/traversals/LCC01Traversal.h#L92-L93

Therefore, I thought it is correct to state that it *should* be the cutoff, even though it is not implemented as such yet?

Is the TODO still accurate? Then I'll open an issue for it.

Reply:
Pretty sure just cutoff would suffice, since this is where we stop iterating over sorted cells. There we use the current particle positions and the additional skin is not needed.Therefore, the comment stands